MAINTAINERS.yml

```diff
@@ -0,0 +1,1178 @@
+        - include/kernel*.h
+        - kernel/
+        - tests/kernel/
+    labels:
+        - "area: Kernel"
+
+Little FS:
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers:
+        - pabigot
+    files:
+        - subsys/fs/Kconfig.littlefs
+        - subsys/fs/littlefs_fs.c
+        - tests/subsys/fs/littlefs
+    description: >-
+       Little FS
+
+Logging:
+    status: maintained
+    maintainers:
+        - nordic-krch
+    files:
+        - include/logging/
+        - samples/subsys/logging/
+        - subsys/logging/
+        - tests/subsys/logging/
+    labels:
+        - "area: Logging"
+
+MAINTAINERS file:
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers:
+        - MaureenHelm
+    collaborators:
+        - ioannisg
+    files:
+        - MAINTAINERS.yml
+    labels:
+        - "area: Process"
+    description: >-
+        Zephyr Maintainers File
+
+MCU Manager:
+    # Provisional entry, subject to approval
+    status: maintained
+    maintainers: de-nordic
+    files:
+        - subsys/mgmt/
+        - include/mgmt/
+        - samples/subsys/mgmt/
+    labels:
+        - "area: mcumgr"
+
+Native POSIX and POSIX arch:
+    status: maintained
+    maintainers:
+        - aescolar
+    files:
+        - arch/posix/
+        - boards/posix/native_posix/
+        - drivers/*/*native_posix*
+        - drivers/*/*/*native_posix*
+        - dts/posix/
+        - include/arch/posix/
+        - scripts/valgrind.supp
+        - soc/posix/
+        - tests/boards/native_posix/
+    labels:
+        - "area: native port"
+    description: >-
+        POSIX architecture and SOC, native_posix board, and related drivers
+
+Networking:
+    status: maintained
+    maintainers:
+        - jukkar
+    collaborators:
+        - tbursztyka
+        - pfalcon
+    files:
+        - drivers/net/
+        - include/net/
+        - samples/net/
+        - subsys/net/
+    files-exclude:
+        - samples/net/sockets/coap_*/
+        - samples/net/lwm2m_client/
+        - subsys/net/lib/coap/
+        - subsys/net/lib/lwm2m/
+        - subsys/net/lib/openthread/
+        - subsys/net/lib/tls_credentials/
+    labels:
+        - "area: Networking"
+
+"Networking: BSD sockets":
+    status: maintained
+    maintainers:
+        - pfalcon
+    files:
+        - samples/net/sockets/
+        - subsys/net/lib/sockets/
+        - tests/net/socket/
+    labels:
+        - "area: Sockets"
+
+"Networking: Buffers":
+    status: maintained
+    maintainers:
+        - jhedberg
+    collaborators:
+        - tbursztyka
+    files:
+        - include/net/buf.h
+        - subsys/net/buf.c
+        - tests/net/buf/
+
+"Networking: CoAP":
+    status: maintained
+    maintainers:
+        - rveerama1
+    files:
+        - subsys/net/lib/coap/
+        - samples/net/sockets/coap_*/
+        - tests/net/lib/coap/
+
+"Networking: LWM2M":
+    status: maintained
+    maintainers:
```

Comment:
@ioannisg LwM2M subsystem now maintained by @rlubos

Reply:
Great, thanks, I updated the entry. Will need, formally, TSC approval.